tuf-spec.md

```diff
@@ -1135,7 +1135,9 @@ non-volatile storage as FILENAME.EXT.
   the new targets metadata file becomes the trusted targets metadata file.
 
   * **4.5**. **Perform a preorder depth-first search for metadata about the
-  desired target, beginning with the top-level targets role.**
+  desired target, beginning with the top-level targets role.**  Note: If
+  metadata requested in steps 4.5.1 - 4.5.2.3 cannot be downloaded nor
+  validated, end the search and report that the target cannot be found.
```

Comment:
This technically covers #17, but can we add "any" in this line?

Note: If **any** metadata requested in steps 4.5.1 - 4.5.2.3 cannot be downloaded nor
validated, end the search and report that the target cannot be found.

I can imagine an implementer seeing the behavior of halting in a sequence of delegations if the first cannot be retrieved as unintended, so emphasis seems called for. Perhaps that's too paranoid.

Reply:
Okay, I revised the text in https://github.com/theupdateframework/specification/pull/24/commits/f57722c05ed1fba04862fefa5225048862e0c168.